src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -455,6 +455,21 @@ public GHTeam createTeam(String name, GHRepository... repositories) throws IOExc
         return createTeam(name, Arrays.asList(repositories));
     }
 
+    /**
+     * Starts a builder that creates a new team.
+     *
+     * <p>
+     * You use the returned builder to set various properties, then call {@link GHCreateTeamBuilder#create()} to finally
+     * create a team.
+     *
+     * @param name
+     *            the name
+     * @return the gh create repository builder
+     */
+    public GHCreateTeamBuilder createTeam(String name) {
+        return new GHCreateTeamBuilder(root, "/orgs/" + login + "/teams", name);
```

Comment:
```suggestion
        return new GHCreateTeamBuilder(root, login, name);
```

Or maybe even this (and add name to builder):
```suggestion
        return new GHCreateTeamBuilder(root);
```
Is a team always created attached to a login?  As me, could I create a team that your login owns?

Reply:
I did this for consistency with the create repo builder, I've made it take an org name now and not the 'apiTailUrl'.

I've kept the name as it's the only mandatory parameter for the builder